libs/ha/ha_common_libs_k8s.py

```diff
@@ -1287,7 +1295,11 @@ def simulate_disk_cvg_failure(self, node_obj, source, resource_status, resource_
         }
         :return: Bool, config_dict/error
         """
-        config_json_file = "config.json"
+        delay = kwargs.get("delay", None)
+        specific_info = kwargs.get("specific_info", None)
```

Comment:
Please change this to this:
specific_info = kwargs.get("specific_info", dict())

Reply:
Changed